src/Corax/Indexing/EntriesModifications.cs

```diff
@@ -212,10 +212,10 @@ private void DeleteAllDuplicates([NotNull] ByteStringContext context)
             //This is made for Set structure.
             if (currentAdd.Equals(currentRemoval))
             {
-                if (Updates.TryPush(currentAdd) == false)
+                if (Updates.TryAdd(currentAdd) == false)
                 {
                     Updates.Grow(context, 1);
-                    Updates.PushUnsafe(currentAdd);
+                    Updates.AddUnsafe(currentAdd);
                 }
```

Comment:
The "growing by one" pattern with the `TryAdd` has a slightly better codegen. We are just checking `Capacity != 0 or Count == Capacity` which is a very fast check. While the other would check if adding will have or not enough space. JIT will emit the fast-path in a very straightforward way without intermediate/inlined calls to `Grow` in the middle. For performance sensitive loops I would still use `TryAdd` instead of the normal `Add` for codegen reasons.